lib/trino-plugin-toolkit/src/test/java/io/trino/plugin/base/security/BaseFileBasedSystemAccessControlTest.java

```diff
@@ -1712,57 +1716,62 @@ public void testViewAuthorization()
         assertAccessDenied(
                 () -> accessControl.checkCanSetEntityAuthorization(user("user", "group", "role"), new EntityKindAndName("VIEW", table), new TrinoPrincipal(ROLE, "new_role")),
                 "Cannot set authorization for view some-catalog.test.table to ROLE new_role");
-
```

Comment:
I think it would be better to extract a method and have to `@Test` methods one per each view kind.

Reply:
Way better to review!